tick/array/tests/serialize_test.py

```diff
@@ -5,12 +5,11 @@
 
 import numpy as np
 import scipy
+import scipy.sparse
```

Comment:
is this import used?

Reply:
yes seems required on windows for some reason

Reply:
Oh, ok maybe should we add a comment. My IDE is likely to remove unused imports and it would be problematic with this one!

Reply:
maybe there's another way around it - it seems odd that it was required but
it fixed the script so I dunno

Reply:
It's ok to me if it is a temporary fix but I think we should leave a comment telling that this is unused but still necessary for windows compilation.
If we just add this comment we can skip travis and appveyor.

Reply:
possible mismatch between windows and linux perhaps?

https://stackoverflow.com/questions/8696322/why-does-this-attributeerror-in-python-occur

should be good now anyway

Reply:
Yes this way it becomes used, perfect and thanks for the link!